docs/usage/configuration-options.md

```diff
@@ -922,6 +922,12 @@ Example for configuring `docker` auth:
 }
 ```
 
+If multiple `hostRules` match a request, then they will be applied in the following order/priority, where latest takes precedence:
+
+- rules with only `hostType` specified
+- rules with only `matchHost` specified (sorted by length of `matchHost` string)
+- rules with both `matchHost` and `hostType` specified (sorted by length of `matchHost` string)
```

Comment:
I think it would be easier to understand this list if it was numbered in order of priority:

1. First rule that matches
2. Second rule that matches
3. Third rule that matches